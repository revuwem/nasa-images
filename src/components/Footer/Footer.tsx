import Logo from "@/components/Logo";
import Paragraph from "@/components/Paragraph";

const Footer = () => {
  return (
    <footer className="pt-8 pb-12 flex flex-wrap gap-6 justify-between items-center border-t border-grey-dark">
      <Logo />
      <Paragraph>
        Made with ğŸ’Ÿ by{" "}
        <a
          href="https://github.com/revuwem"
          target="_blank"
          className="text-grey-light hover:text-grey-dark"
        >
          revuwem
        </a>
      </Paragraph>
    </footer>
  );
};

export default Footer;
