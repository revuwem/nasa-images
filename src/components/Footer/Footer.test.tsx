import { describe, it } from "vitest";
import renderer from "react-test-renderer";
import { BrowserRouter } from "react-router-dom";
import Footer from "@/components/Footer";

describe("CollectionCard", () => {
  it("should render correctly", () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <Footer />
        </BrowserRouter>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
