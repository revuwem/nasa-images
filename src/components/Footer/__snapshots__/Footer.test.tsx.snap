// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`CollectionCard > should render correctly 1`] = `
<footer
  className="pt-8 pb-12 flex flex-wrap gap-6 justify-between items-center border-t border-grey-dark"
>
  <a
    className="group"
    href="/"
    onClick={[Function]}
  >
    <div
      className="flex gap-1 items-center"
    >
      <img
        alt=""
        className="w-14 h-14"
        src="/nasa.svg"
      />
      <h1
        className="text-grey-light font-bold text-xl transition group-hover:text-grey-dark"
      >
        NASA Images
      </h1>
    </div>
  </a>
  <p
    className="text-base text-grey-dark"
  >
    Made with ðŸ’Ÿ by
     
    <a
      className="text-grey-light hover:text-grey-dark"
      href="https://github.com/revuwem"
      target="_blank"
    >
      revuwem
    </a>
  </p>
</footer>
`;
